include/picongpu/simulation_defines/param/ionizer.param

```diff
@@ -423,6 +423,20 @@ namespace thomasFermi
     constexpr float_X TFC1    = -0.366667;
     constexpr float_X TFC2    =  0.983333;
 
+    /** cutoff energy for electron "temperature" calculation
+     *
+     * In laser produced plasmas we can have different, well-separable groups
+     * of electrons. For the Thomas-Fermi ionization model we only want the
+     * thermalized "bulk" electrons. Including the high-energy "prompt"
```

Comment:
missing subject: add "[...] prompt *electrons*"

Reply:
this new cut-off and the upcoming one should in included in the model description of the `docs/`.

You can e.g. just take a variant of your good PR description.

Reply:
I added a description in `/docs` and also the missing subject has been added - just on a new line which is why the comment won't vanish. :smile: